x-pack/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -20,7 +21,7 @@ import { navTabs } from './home_navigations';
 import { useSignalIndex } from '../../detections/containers/detection_engine/alerts/use_signal_index';
 
 const WrappedByAutoSizer = styled.div`
-  height: 100%;
+  min-height: calc(100vh - 48px);
```

Comment:
This whole styling might end up being removed if we decide to go with EuiFlexGroup and EuiFlexItem.